bin/run_tests

```diff
@@ -27,28 +27,35 @@ PATH=$PROJECT_DIR/bin:$PATH
 # Arguments to trigger the tests
 DIRECT_MODEL_TESTS=0
 PROXY_MODEL_TESTS=0
+DIRECT_MODEL_SGX_TESTS=0
 
 main() {
 
     if [ "$DIRECT_MODEL_TESTS" == "1" ]; then
         run_docker_test ./ci/avalon-direct-model-tests.yaml
     elif [ "$PROXY_MODEL_TESTS" == "1" ]; then
         run_docker_test ./ci/avalon-proxy-model-tests.yaml
+    elif [ "$DIRECT_MODEL_SGX_TESTS" == "1" ]; then
+	run_docker_test ./ci/avalon-direct-model-sgx-tests.yaml
     fi
 }
 
-while getopts "dph" opt; do
+while getopts "dpsh" opt; do
     case $opt in
         d)
             DIRECT_MODEL_TESTS=1
             ;;
         p)
             PROXY_MODEL_TESTS=1
             ;;
+	s)
```

Comment:
Why do we need this option? There is no difference in testing with respect to HW/SIM mode.

Reply:
there is additional volume map and device map in case of SGX HW mode for avalon-enclave-manager service